### ISSUE TITLE
[processor/servicegraphprocessor] Not included in the `v0.61.0` release image

### ISSUE THREAD

**Issue by sherifkayad** [open-telemetry/opentelemetry-collector-contrib#14899]
## Description
It seems like the Service Graph Processor (https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/servicegraphprocessor) isn't included in the `contrib` distro. Is that by mistake? Was that intentional?

I think it should be included as it's officially released in the contrib source code since `v0.60.0` (https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.60.0)

With image version 0.61.0 of the contrib collector I am getting:
```
collector server run finished with error: failed to get config: cannot unmarshal the configuration: unknown processors type \"servicegraph\" for \"servicegraph\" (valid values: [cumulativetodelta resourcedetection resource memory_limiter attributes k8sattributes experimental_metricsgeneration probabilistic_sampler spanmetrics transform batch deltatorate filter groupbytrace metricstransform span groupbyattrs redaction routing tail_sampling])\n
```

## Expected Result
The `servicegraph` processor is available and can be configured

### Collector version

v0.61.0

## Environment
Docker / Kubernetes

### Additional context

I also filed https://github.com/open-telemetry/opentelemetry-collector-releases/issues/217 as I wasn't sure where the issue should be addressed

**Comment by mapno** [comment]
I missed this when I first merged the processor. Thanks for reporting. Opened this PR to included in future builds: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14932

**Comment by fatsheep9146** [comment]
I think this issue is solved by #14392.
